torch/nn/functional.py

```diff
@@ -795,6 +809,13 @@ def binary_cross_entropy_with_logits(input, target, weight=None, size_average=Tr
                 over observations for each minibatch. However, if the field
                 sizeAverage is set to False, the losses are instead summed
                 for each minibatch. Default: True
+
```

Comment:
http://pytorch.org/docs/master/nn.html#torch.nn.functional.nll_loss

This work is motivated by the link above. Both `F.nll_loss` and `nn.NLLLoss` has examples. I think it would be nice to have the examples in both places to help understading.